### PR TITLE
add :select-drawmode method to camera-model

### PR DESCRIPTION
**Pull request by YoheiKakiuchi**
This PR is camera-model version of https://github.com/euslisp/jskeus/pull/308

### CODE REVIEW

**Comment by k-okada** [comment]
Please add sample code 


--
◉ Kei Okada

2015/11/23 21:10、Yohei Kakiuchi <notifications@github.com> のメッセージ:

**Comment by YoheiKakiuchi** [comment]
Add code for generating hidden-line image to demo program.

hidden-line-camera-image

File: irteus/irtsensor.l [hunk 1]
```diff
@@ -356,6 +356,21 @@
            )))
      img
      ))
+  (:select-drawmode (mode objs)
```
Comment (k-okada) [inline irteus/irtsensor.l]: please add documentation

Comment (k-okada) [inline irteus/irtsensor.l]: if this is 'public' method

Comment (YoheiKakiuchi) [inline irteus/irtsensor.l]: add document

**Pull request merged**